src/inverseRobot.js

```diff
@@ -7,7 +7,27 @@
 */
 
 function inverseRobot(robot) {
-  // write code here
+  const fixedRobot = {};
+
+  const keys = Object.keys(robot);
+
+  const values = Object.values(robot);
+
+  for (let i = 0; i < keys.length; i++) {
+    const checkKeys = keys.indexOf(keys[i], i + 1);
+
+    const checkValues = values.indexOf(values[i], i + 1);
+
+    if (checkKeys > -1 || checkValues > -1) {
+      return null;
+    }
+  }
+
+  for (let i = 0; i < values.length; i++) {
+    fixedRobot[values[i]] = keys[i];
+  }
+
+  return fixedRobot;
```

Comment:
This solution is too complicated. You can do it much simpler way by using one loop and calling robot's keys and values directly without converting to an arrays